pep-0726.rst

```diff
@@ -134,9 +134,42 @@ module ``__dict__``.  If present, the appropriate function is called to
 customize setting the attribute or its deletion, else the normal
 mechanism (storing/deleting the value in the module dictionary) will work.
 
-Defining ``__setattr__`` or ``__delattr__`` only affect lookups made
-using the attribute access syntax---directly accessing the module
-globals is unaffected, e.g. ``sys.modules[__name__].some_global = 'spam'``.
+Defining module ``__setattr__`` or ``__delattr__`` only affect lookups made
+using the attribute access syntax --- directly accessing the module globals
+(whether by globals() within the module, or via a reference to the module’s
+globals dictionary) is unaffected.  For example:
+
+.. code:: pycon
+
+  >>> import mod
+  >>> mod.__dict__['foo'] = 'spam'  # bypasses __setattr__, defined in mod.py
+
+or
+
+.. code:: python
+
+  # mod.py
+
+  def __setattr__(name, value):
+      ...
+
+  foo = 'spam'  # bypasses __setattr__
+  globals()['bar'] = 'spam'  # here too
```

Comment:
Presumably also:
```python
def function():
    global foo
    foo = 'baz'  # also bypasses __setattr__
```

Reply:
> I just think it is worth listing the different ways

I'm worrying about a combinatorial grows of the number of such ways... :)

E.g.
>  a function is that it could be called after the module has finished being imported

Or while module was loaded.  Before or after the ``__setattr__`` definition...

But in fact the only difference is in using attribute access syntax (by ``mod.foo`` or  ``sys.modules[__name__].foo`` within the module) vs all other cases directly accessing the module globals or the module's globals dictionary).

> Maybe that makes no difference or maybe it does make a difference.

Given above it should be clear the answer is "no".

So, I would appreciate another opinion.

Reply:
I think the global statement is different enough that it's worth adding an explicit example.